src/i_winmusic.c

```diff
@@ -713,7 +814,7 @@ static void I_WIN_SetMusicVolume(int volume)
 
     volume_factor = sqrtf((float)volume / 15);
 
-    update_volume = true;
+    update_volume = (song.file != NULL);
```

Comment:
Why is this change?

Reply:
The channel volume is already initially updated in `ResetDevice()` before every song now. It's placed there so that the optional delay can be the last message that's sent after all the reset messages finish. If this is left at true, volume is reset twice for each channel. There's no need to set this before a song has loaded.